types/activex-excel/index.d.ts

```diff
@@ -3330,18 +3507,70 @@ declare namespace Excel {
         readonly Hwnd: number;
         IgnoreRemoteRequests: boolean;
         InchesToPoints(Inches: number): number;
-        InputBox(Prompt: string, Title?: string, Default?: string, Left?: number, Top?: number, HelpFile?: string, HelpContextID?: number, Type?: InputBoxReturnType): string | number | boolean | Range | Error | SafeArray<string> | SafeArray<number> | SafeArray<boolean>;
+        InputBox(
+            Prompt: string,
+            Title?: string,
+            Default?: string,
+            Left?: number,
+            Top?: number,
+            HelpFile?: string,
+            HelpContextID?: number,
+            Type?: InputBoxReturnType
+        ): string | number | boolean | Range | Error | SafeArray<string> | SafeArray<number> | SafeArray<boolean>;
         Interactive: boolean;
         International: SafeArray & (<T>(Index: T) => InternationalResult<T>);
-        Intersect(Arg1: Range, Arg2: Range, Arg3?: Range, Arg4?: Range, Arg5?: Range, Arg6?: Range, Arg7?: Range, Arg8?: Range, Arg9?: Range, Arg10?: Range, Arg11?: Range, Arg12?: Range, Arg13?: Range, Arg14?: Range, Arg15?: Range, Arg16?: Range, Arg17?: Range, Arg18?: Range, Arg19?: Range, Arg20?: Range, Arg21?: Range, Arg22?: Range, Arg23?: Range, Arg24?: Range, Arg25?: Range, Arg26?: Range, Arg27?: Range, Arg28?: Range, Arg29?: Range, Arg30?: Range): Range;
+        Intersect(
+            Arg1: Range,
+            Arg2: Range,
+            Arg3?: Range,
+            Arg4?: Range,
+            Arg5?: Range,
+            Arg6?: Range,
+            Arg7?: Range,
+            Arg8?: Range,
+            Arg9?: Range,
+            Arg10?: Range,
+            Arg11?: Range,
+            Arg12?: Range,
+            Arg13?: Range,
+            Arg14?: Range,
+            Arg15?: Range,
+            Arg16?: Range,
+            Arg17?: Range,
+            Arg18?: Range,
+            Arg19?: Range,
+            Arg20?: Range,
+            Arg21?: Range,
+            Arg22?: Range,
+            Arg23?: Range,
+            Arg24?: Range,
+            Arg25?: Range,
+            Arg26?: Range,
+            Arg27?: Range,
+            Arg28?: Range,
+            Arg29?: Range,
+            Arg30?: Range
+        ): Range;
```

Comment:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/35958/files#r290613779 Or even if the arguments are all the same type?